app.yml

```diff
@@ -67,70 +67,48 @@ default_permissions:
 
   # Issues and related comments, assignees, labels, and milestones.
   # https://developer.github.com/v3/apps/permissions/#permission-on-issues
-  issues: write
+  # issues: read
 
   # Search repositories, list collaborators, and access repository metadata.
   # https://developer.github.com/v3/apps/permissions/#metadata-permissions
-  # metadata: read
+  metadata: read
 
   # Retrieve Pages statuses, configuration, and builds, as well as create new builds.
   # https://developer.github.com/v3/apps/permissions/#permission-on-pages
   # pages: read
-
   # Pull requests and related comments, assignees, labels, milestones, and merges.
   # https://developer.github.com/v3/apps/permissions/#permission-on-pull-requests
   # pull_requests: read
-
   # Manage the post-receive hooks for a repository.
   # https://developer.github.com/v3/apps/permissions/#permission-on-repository-hooks
   # repository_hooks: read
-
   # Manage repository projects, columns, and cards.
   # https://developer.github.com/v3/apps/permissions/#permission-on-repository-projects
   # repository_projects: read
-
   # Retrieve security vulnerability alerts.
   # https://developer.github.com/v4/object/repositoryvulnerabilityalert/
   # vulnerability_alerts: read
-
   # Commit statuses.
   # https://developer.github.com/v3/apps/permissions/#permission-on-statuses
   # statuses: read
-
   # Organization members and teams.
   # https://developer.github.com/v3/apps/permissions/#permission-on-members
   # members: read
-
   # View and manage users blocked by the organization.
   # https://developer.github.com/v3/apps/permissions/#permission-on-organization-user-blocking
   # organization_user_blocking: read
-
   # Manage organization projects, columns, and cards.
   # https://developer.github.com/v3/apps/permissions/#permission-on-organization-projects
   # organization_projects: read
-
   # Manage team discussions and related comments.
   # https://developer.github.com/v3/apps/permissions/#permission-on-team-discussions
   # team_discussions: read
-
   # Manage the post-receive hooks for an organization.
   # https://developer.github.com/v3/apps/permissions/#permission-on-organization-hooks
   # organization_hooks: read
-
   # Get notified of, and update, content references.
   # https://developer.github.com/v3/apps/permissions/
   # organization_administration: read
 
-
-# The name of the GitHub App. Defaults to the name specified in package.json
-# name: My Probot App
-
-# The homepage of your GitHub App.
-# url: https://example.com/
-
-# A description of the GitHub App.
-# description: A description of my awesome app
-
 # Set to true when your GitHub App is available to the public or false when it is only accessible to the owner of the app.
-# Default: true
-# public: false
+public: false
```

Comment:
Ahh, good that we can configure this in the manifest.

Reply:
Yep!

Also turns out this is just a 1:1 mapping of the form inputs on https://github.com/settings/apps/new.

Still need to check that the following actually means an "internal" app like described elsewhere.

![image](https://user-images.githubusercontent.com/51677/48188780-74ac7b00-e336-11e8-9947-456b70448fbc.png)

Reply:
Yay!